media-sound/liblc3/liblc3-1.0.3.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="LC3 is an efficient low latency audio codec."
+HOMEPAGE="https://github.com/google/liblc3"
+SRC_URI="https://github.com/google/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${PN}-v${PV}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="tools"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86"
```

Comment:
See https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages.

Reply:
OK, will only leave amd64 in there then.